go/vt/servenv/buildinfo.go

```diff
@@ -94,7 +94,7 @@ func (v *versionInfo) MySQLVersion() string {
 	if mySQLServerVersion != "" {
 		return mySQLServerVersion
 	}
-	return "5.7.9-vitess-" + v.version
+	return "8.0.31-vitess-" + v.version
```

Comment:
I think we also need to change https://github.com/vitessio/vitess/blob/ece6501975604ef5d0be3ab256145047e13d9c00/go/mysql/collations/local.go#L38-L44 together with this?

Otherwise we get a reported version of 8.0, but still default collation settings for MySQL 5.7?

The code might warrant a refactor there so that we don't have to update this in multiple places somehow?

Reply:
done.